pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -125,16 +125,24 @@ type NodeTopologyMap struct {
 	Labels map[string]TopologyLabelValues `json:"labels,omitempty"`
 }
 
-// ConditionReconcileComplete communicates the status of the StorageCluster resource's
-// reconcile functionality. Basically, is the Reconcile function running to completion.
-const ConditionReconcileComplete conditionsv1.ConditionType = "ReconcileComplete"
+const (
+	// ConditionReconcileComplete communicates the status of the StorageCluster resource's
+	// reconcile functionality. Basically, is the Reconcile function running to completion.
+	ConditionReconcileComplete conditionsv1.ConditionType = "ReconcileComplete"
+
+	// ConditionExternalClusterConnected condition type indicates
+	// the successful connection to an external cluster
+	ConditionExternalClusterConnected conditionsv1.ConditionType = "ExternalClusterConnected"
+)
 
 // List of constants to show different different reconciliation messages and statuses.
 const (
-	ReconcileFailed           = "ReconcileFailed"
-	ReconcileInit             = "Init"
-	ReconcileCompleted        = "ReconcileCompleted"
-	ReconcileCompletedMessage = "Reconcile completed successfully"
+	ReconcileFailed                 = "ReconcileFailed"
+	ReconcileInit                   = "Init"
+	ReconcileCompleted              = "ReconcileCompleted"
+	ReconcileCompletedMessage       = "Reconcile completed successfully"
+	ExternalClusterConnected        = "ExternalClusterConnected"
```

Comment:
This is a Phase. There is a separate place for this.

Reply:
```suggestion
	ExternalClusterConnected        = "Connected"
```

Reply:
@umangachapagain , What I added was a condition, not a phase... Do you still think, we need phase here not condition?